gm4_combat_expanded/data/gm4_combat_expanded/loot_tables/armor/identification/modifier/vamp.json

```diff
@@ -13,6 +13,7 @@
               "modifiers": [
                 {
                   "attribute": "minecraft:generic.armor",
+                  "replace": false,
```

Comment:
This is not where `replace` goes. You need to put it at the same level as `function` and `modifiers`.

Reply:
fixed this, did not fix the bug however